priv/perf/apps/load_test/lib/ethereum/ethereum.ex

```diff
@@ -49,6 +49,23 @@ defmodule LoadTest.Ethereum do
     end
   end
 
+  @spec get_gas_used(String.t()) :: non_neg_integer()
+  def get_gas_used(receipt_hash) do
+    result =
+      {Ethereumex.HttpClient.eth_get_transaction_receipt(receipt_hash),
+       Ethereumex.HttpClient.eth_get_transaction_by_hash(receipt_hash)}
+
+    case result do
+      {{:ok, %{"gasUsed" => gas_used}}, {:ok, %{"gasPrice" => gas_price}}} ->
+        {gas_price_value, ""} = gas_price |> String.replace_prefix("0x", "") |> Integer.parse(16)
+        {gas_used_value, ""} = gas_used |> String.replace_prefix("0x", "") |> Integer.parse(16)
+        gas_price_value * gas_used_value
+
+      {{:ok, nil}, {:ok, nil}} ->
```

Comment:
Doesn't it break something when we return that transaction used 0 gas?
In what cases receipt and hash are nil?

Reply:
For now, I removed `case`, this function is copied from cabbage tests